markdown/dev/guides/writers/editing-strapi-files/en.md

```diff
@@ -0,0 +1,11 @@
+---
+title: How to edit Strapi files
```

Comment:
There will be no more strapi in v3. You can drop this. Or certainly don't have to invest more time in it.